service/controller/app/v1/resource/chartoperator/create.go

```diff
@@ -21,6 +22,15 @@ func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	if key.InCluster(cr) {
+		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("app %#q is targeting a control plance, no need to setup a chart operator", cr.Name))
+
+		resourcecanceledcontext.SetCanceled(ctx)
```

Comment:
Canceling like this is okay? or just return nil at this point?

Reply:
https://github.com/giantswarm/operatorkit/blob/master/docs/control_flow_primitives.md#cancel-resources

Reply:
> The resources being configured after the canceled resource are then executed and the canceled resource is executed again on the next reconciliation loop once the resync period kicks in again.
I will remove this line then, Thanks.